### DISCUSSION TITLE
Please tell me your favorite number

### DISCUSSION THREAD

**Discussion by westvally** [NAIST-SE/AttractingContributors#143] [General]
https://github.com/eIectro/AttractingContributors/blob/main/README.md

**Comment by zhangyongxin1212** [comment]
Added, plz help me too
https://github.com/zhangyongxin1212/AttractingContributors-Game

**Reply by westvally** [reply]
Thank you.
I added.

**Comment by nagata-atsuki** [comment]
added!!!!
Help me!!!!!
https://github.com/nagata-atsuki/AttractingContributors-Athlete

**Reply by westvally** [reply]
thank you.
I did.

**Comment by ellielo** [comment]
8

**Comment by YPanY** [comment]
added 7, pls help me by adding your favorite programming language or framework here👇 THX!
https://github.com/Pan-NLP-NAIST/AttractingContributors-programming_language/blob/main/README.md

**Comment by t-matsumoto4758** [comment]
added! please help me!
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown
